src/py/aspen/cli/db.py

```diff
@@ -108,3 +128,115 @@ def import_covidhub_project(
         s3_dst_prefix,
         auth0_usermap,
     )
+
+
+@db.command("create-phylo-run")
+@click.option(
+    "--group-name",
+    type=str,
+    required=True,
+    help="Name of the group to create the phylo run under.",
+)
+@click.option("--all-group-sequences", is_flag=True, required=False, default=False)
+@click.option("samples", "--sample", type=str, required=False, multiple=True)
+@click.option("--aligned-gisaid-dump-id", type=int, required=False, default=None)
+@click.option(
+    "--builds-template-file",
+    type=str,
+    required=False,
+    default="workflows/setup_nextstrain_run/builds_template.yaml",
+)
+@click.option("--builds-template-args", type=str, required=True)
+@click.pass_context
+def create_phylo_run(
+    ctx,
+    group_name,
+    all_group_sequences,
+    samples,
+    aligned_gisaid_dump_id,
+    builds_template_file,
+    builds_template_args,
+):
+    # these are injected into the IPython scope, but they appear to be unused.
+    engine = ctx.obj["ENGINE"]
+
+    with session_scope(engine) as session:
+        group = session.query(Group).filter(Group.name == group_name).one()
+
+        all_samples: Iterable[Sample] = (
+            session.query(Sample)
+            .filter(
+                or_(
+                    Sample.public_identifier.in_(samples),
+                    and_(
+                        expression.true()
```

Comment:
What's getting checked here?

Reply:
A sample is included if one of the following is true:

1. The sample is explicitly requested using `--sample`
2. `--all-group-samples` is specified AND the sample belongs to the group.